templates/server.conf.j2

```diff
@@ -10,9 +10,9 @@ key {{openvpn_key_dir}}/server.key
 dh {{openvpn_key_dir}}/dh.pem
 {% if tls_auth_required %}
 tls-auth {{openvpn_key_dir}}/ta.key 0
+tls-server
```

Comment:
`tls-auth` is a bit of a bad naming. If we enable it, that means we sign each packet, not that TLS is enabled.

`tls-server` is required since we're acting as a server - https://openvpn.net/index.php/open-source/documentation/manuals/65-openvpn-20x-manpage.html#lbAK

Could you revert this specific change?